test/musica/micm/CMakeLists.txt

```diff
@@ -1,21 +1,17 @@
-find_package(PkgConfig REQUIRED)
-pkg_check_modules(netcdff IMPORTED_TARGET REQUIRED netcdf-fortran)
-
 find_package(musica ${MUSICA_VERSION} REQUIRED)
 message(STATUS "  MUSICA version ${MUSICA_VERSION}")
 
 include(FetchContent)
 
 FetchContent_Declare(musica
   GIT_REPOSITORY https://github.com/NCAR/musica.git
-  GIT_TAG        8ab0aa0
+  GIT_TAG        main
```

Comment:
I feel like this should probably be a hash instead of the head of "main".  I acknowledge this is a maintenance burden, but I believe we need to be able to to control when we move externals to newer versions

Reply:
Fixed